apps/full-site-editing/full-site-editing-plugin/posts-list-block/templates/post-item.php

```diff
@@ -31,9 +31,11 @@
 		<span class="a8c-posts-list-item__author"><?php echo esc_html_x( 'by', 'designating the post author (eg: by John Doe', 'full-site-editing' ); ?>
 			<?php the_author_posts_link(); ?>
 		</span>
+		<?php if ( current_user_can( 'edit_posts' ) ) : ?>	
 		<span class="a8c-posts-list-item__edit-link">
```

Comment:
I feel we should indent here one level. Or not? I'm not super familiar with PHP/HTML mix rules so leaving decision up to you